### ISSUE TITLE
Kubernetes v1.22+ compatibility

### ISSUE THREAD

**Issue by Subetov** [Orange-OpenSource/casskop#385]
## Feature Request

**Is your feature request related to a problem? Please describe.**
Helm charts compatibility with kubernetes v1.22+

**Describe the solution you'd like to see**
Need to fix manifest with API versions:
- authorization.k8s.io/v1beta1
- apiextensions.k8s.io/v1beta1
- rbac.authorization.k8s.io/v1beta1

**Comment by cscetbon** [comment]
@Subetov there is a pending PR that upgrades the framework used for the operator. I suppose it'll solve this issue. @AKamyshnikova can you confirm ? If not, it'll be better to wait after #376 is approved and merged.

**Comment by AKamyshnikova** [comment]
@cscetbon @Subetov Yes, upgrading operator-sdk unlocks upgrade/compatibility  with kuberentes versions [1]. At this moment PR #376 updates client-go to v0.20.2 version [2]
[1] - https://sdk.operatorframework.io/docs/overview/#kubernetes-version-compatibility
[2] - https://github.com/kubernetes/client-go#compatibility-matrix

**Comment by kzzalews** [comment]
As #376 was merged and version 2.1.0 of CassKop operator was released two days ago, do we know which versions of K8s it supports right now? Am I right that we've just gained support of K8s 1.22? Is migration of old `CassandraCluster` objects necessary? For sure, I would like to test it, but getting such info from author @AKamyshnikova will be much appreciated :)

**Comment by AKamyshnikova** [comment]
@kzzalews Sorry, in #376 I was not able to update client-go to v.0.22.0 in the end https://github.com/Orange-OpenSource/casskop/blob/master/go.mod#L103 - the was an issue with unit test.
So current issue is still actual and required separate pull request.
No additional steps are required for an update.

**Comment by cscetbon** [comment]
Also fyi be careful that the release i did failed on releasing the helm chart. @fdehay is fixing it.
